app/views/users/_publishing_from_rss.html.erb

```diff
@@ -1,4 +1,17 @@
 <h3>Publishing to dev.to from RSS</h3>
+<% if @user.feed_url.present? %>
+  <%= form_for(@user) do |f| %>
+    <div class="field">
+      <label></label>
+      <%= f.hidden_field :feed_url, value: @user.feed_url %>
+      <%= f.hidden_field :tab, value: @tab %>
+      <%= f.submit "Re-Fetch" %>
```

Comment:
What about "Refresh feed" or "Fetch again" ? Do you think it's less self explanatory than "Re-Fetch"?

Reply:
i think "Refresh Feed" works better